src/main/groovy/io/openliberty/tools/gradle/tasks/UndeployTask.groovy

```diff
@@ -79,8 +79,7 @@ class UndeployTask extends AbstractServerTask {
             File serverXML = new File(getServerDir(project).getCanonicalPath(), "server.xml")
 
             try {
-                getServerConfigDocument(new CommonLogger(project), serverXML, server.configDirectory,
-                        server.bootstrapPropertiesFile, combinedBootstrapProperties, server.serverEnvFile, false, getLibertyDirectoryPropertyFiles(null))
+                scd= getServerConfigDocument(new CommonLogger(project), getLibertyDirectoryPropertyFiles(null))
```

Comment:
I don't think you need to get serverXML at line 79 any more.

Reply:
Added back logic in getServerDocument() to compare server.xml